src/components/tabs/hooks/use-tab-items.ts

```diff
@@ -94,16 +94,20 @@ function validateTabChildren(
   initialActiveIndex: number
 ): RawTabItem[] {
   /**
-   * Disallow rendering a `Tabs` component with only one child.
+   * Warn if rendering a `Tabs` component with only one child.
```

Comment:
For this case, was it just that one tab rendered a single codeblock? That _seems_ like a reasonable usecase, no?